index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mens Portfolio</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+
+<div class="flex_container">
+    <nav class="flex_nav">
+        <a href="" class="toggleNav"> 
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </a>
+        <ul>
+            <li class=""><a href="#">Hello</a></li>
+            <li class=""><a href="#">Portfolio</a></li>
+            <li class=""><a href="#">About</a></li>
+            <li class=""><a href="#">Contact</a></li> 
+        </ul>
+    </nav>

```

Comment:
As of the [project's requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#project-requirements), we have to build two sections. Let's include the `nav` inside the `header` tag so that we can achieve this project's requirement and also follow proper [HTML5 semantic tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#html--css-best-practices). Kindly update that.